notebooks/book2/17/gpKernelPlot.ipynb

```diff
@@ -0,0 +1,369 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def gpKernelPlot(seed):
</pre><p>Use PEP8 names for methods. <code>gp_kernel_plot</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Done